src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/tests/index.cypress.spec.js

```diff
@@ -35,6 +35,10 @@ const setup = ({ authenticated, isCerner } = {}) => {
 };
 
 describe('The schedule view VA appointments page', () => {
+  before(function() {
+    if (Cypress.env('CIRCLECI')) this.skip();
+  });
```

Comment:
@department-of-veterans-affairs/vsa-public-websites-frontend: FYSA, we are still trying to move to Circle eventually as our primary CI/CD service, which throws a wrench in the pile of anything related to content, including E2E tests for static pages.

We're still discussing what this will mean in the future. So no immediate action required on your part. Just wanted to let you know that Cypress tests are likely to remain unsupported for static pages once we transition.

My initial, unverified suggestion is that maybe things like CTA widgets can be tested in a "UI test" kind of way with React Testing Library, but I'll leave it up to you all. 🤷